### ISSUE TITLE
A bug when change the loss function

### ISSUE THREAD

**Issue by LILIXIYA** [XuecaiHu/Meta-SR-Pytorch#62]
Hi, 

I think the code has some bugs when using the GAN loss. I changed the loss to 0.5*L1+0.5*GAN, but it reports some error message. Have you tested the code using GAN loss? Thanks!
